searx/engines/emojipedia.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+"""Emojipedia
+
+Emojipedia is an emoji reference website which documents the meaning and
+common usage of emoji characters in the Unicode Standard.  It is owned by Zedge
+since 2021. Emojipedia is a voting member of The Unicode Consortium.[1]
+
+[1] https://en.wikipedia.org/wiki/Emojipedia
+"""
+
+from urllib.parse import urlencode
+from lxml import html
+
+from searx.utils import (
+    eval_xpath_list,
+    eval_xpath_getindex,
+    extract_text,
+)
+
+about = {
+    "website": 'https://emojipedia.org',
+    "wikidata_id": 'Q22908129',
+    "official_api_documentation": None,
+    "use_official_api": False,
+    "require_api_key": False,
+    "results": 'HTML',
+}
+
+categories = ['general']
```

Comment:
`categories = []` seems better, since emojipedia won't return answers like the engines in the general category.

Reply:
> since emojipedia won't return answers like the engines in the general category.

@dalf @tiekoetter : Sure? .. IMO it is a reference book, like wikipedia .. ["Emojipedia is an emoji reference website"](https://en.wikipedia.org/wiki/Emoji)  .. it describes the meaning and common usage of emoji characters in the [Unicode Standard](https://en.wikipedia.org/wiki/Unicode_Standard).

Reply:
Wikipedia covers a wide range of topics, Emojipedia is dedicated to emoji and nothing else.
I think engines in the general category should cover a wide range of topics, otherwise they belongs to another category.

Reply:
ping @return42

Reply:
I am very sorry for the long delay on fixing this trivia :-) .. done https://github.com/searxng/searxng/compare/1a72b30063bb4944fca1b2e6ce56471857776a40..445cba7c0294547af7bf688007bd9b29dee058e3